config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  root to: 'high_voltage/pages#show', id: 'home'
+  root to: 'high_voltage/pages#show', id: 'landing'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.